CHANGELOG.rst

```diff
@@ -17,7 +17,20 @@ Internal Changes
 
 **Improvements**
 
-* **Infrastructure Components**: ``redis`` has been upgraded from version ``7.0.7`` to ``7.4.1``
+- **Infrastructure Components**: ``redis`` has been upgraded from version ``7.0.7`` to ``7.4.1``
+- **Search services**: Add support for sentry
+
+
+**Bug Fixes**
+
+- **Search services**: Don't return results without linked namespaces
+
+
+Individual Components
+~~~~~~~~~~~~~~~~~~~~~
+
+- `renku-search 0.7.0 <https://github.com/SwissDataScienceCenter/renku-search/releases/tag/v0.7.0>`_
+
 
 * **Helm chart**: remove the custom-made Gitlab Omnibus Helm chart from Renku dependencies
```

Comment:
This should be up in the "Improvements" section I think?

Reply:
oh yes, sorry!